client/my-sites/site-settings/composing/publish-confirmation.jsx

```diff
@@ -43,7 +46,24 @@ class PublishConfirmation extends Component {
 	}
 
 	render() {
-		const { fetchingPreferences, translate } = this.props;
+		const { fetchingPreferences, translate, showPublishFlow } = this.props;
+
+		if ( showPublishFlow ) {
+			return (
+				<FormFieldset>
+					<FormLabel>{ translate( 'Show publish confirmation' ) }</FormLabel>
+					<FormSettingExplanation isIndented>
+						{ translate(
+							'The confirmation step for double-checking your content before publishing ' +
+								'now is handled by the new block editor.' +
+								'To show the publish confirmation with the block editor, ' +
```

Comment:
@Automattic/editorial When you get a chance, would you mind looking over this description?

Reply:
Thanks, Konstantin

Reply:
@Automattic/editorial or maybe @michelleweber , could we ask you for input on the content here?